src/dftbp/common/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ set(curdir ${CMAKE_CURRENT_SOURCE_DIR})
 
 set(sources-fpp
   ${curdir}/accuracy.F90
+  ${curdir}/append.F90
```

Comment:
Possibly matrixappend.F90 would be clearer?

Reply:
I think so, changed by now.